envoy-control-tests/src/main/resources/oauth/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine
+RUN apk add openjdk11-jre git
+RUN git clone --depth 1 --branch 0.0.3 https://github.com/Kornelos/oauth-mock.git
```

Comment:
I missed this, what I meant was transferring the whole thing into this repository or moving it from `Kornelos/oauth-mock` to `allegro/oauth-mock`.